parser/parsetypes/parsetypes.go

```diff
@@ -33,6 +33,10 @@ func NewBroDataFactory(fileType string) func() BroData {
 		return func() BroData {
 			return &HTTP{}
 		}
+	} else if strings.HasPrefix(fileType, "open") {
```

Comment:
Might go with "open_conn" in case someone has a weird plugin that also uses "openXXXX" for a log name.
The other logs are default included in Zeek, so there is less opportunity for collisions.

Reply:
yep, sounds good!